src/components/navButtons.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div>
+    <!-- top nav buttons -->
+    <div class="top-nav-btns" v-show="this.topType === 'default-light'">
+      <img src="../assets/icons/arrow-left-light.svg" alt />
+    </div>
+    <div class="top-nav-btns" v-show="this.topType === 'default-dark'">
+      <img src="../assets/icons/arrow-left-dark.svg" alt />
+    </div>
+    <div class="top-nav-btns" v-show="this.topType === 'share-light'">
+      <img src="../assets/icons/arrow-left-light.svg" alt />
+      <img src="../assets/icons/share-light.svg" alt />
+    </div>
+    <div class="top-nav-btns" v-show="this.topType === 'share-dark'">
+      <img src="../assets/icons/arrow-left-dark.svg" alt />
+      <img src="../assets/icons/share-dark.svg" alt />
+    </div>
+
+    <!-- bottom nav buttons -->
+    <div class="bot-nav-btns" v-show="this.botType === 'next-only'">
```

Comment:
Maybe make all the images `cursor: pointer` or some hover anims (we need mixins for this because we're gonna use them so so often) so desktop users to show that the buttons are clickable

Reply:
yep this is still a very basic but functional component